addon/webextension/background/communication.js

```diff
@@ -18,6 +18,7 @@ window.communication = (function () {
     let func = registeredFunctions[req.funcName];
     let result;
     try {
+      req.args.push(sender);
```

Comment:
I'm gonna move `sender` to be the first argument. The `register`ed callback will change signature to `(sender, ...args)`